shared/common-adapters/input.native.js

```diff
@@ -178,6 +179,8 @@ class Input extends Component<Props, State> {
   _lineHeight = () => {
     if (this.props.small) {
       return 20
+    } else if (isAndroid) {
```

Comment:
this will affect all inputs. desired?

Reply:
Good call. Single-line inputs looked okay, but my intention is to fix large, multiline inputs on Android.